utils/lists.ts

```diff
@@ -337,6 +337,11 @@ export const derivativesList: Record<string, string>[] = [
     description: "Cyberpunk alternate Loot universe",
     url: "https://www.lootmetaverse.club/",
   },
+  {
+    name: "Loot (for Sex)",
+    description: "Sex themed loot for adventurers, punks, art etc...",
```

Comment:
Can you remove the `...` at the end of description

Reply:
Of course not a problem! I just assumed that’s how etc… was written, I fixed the file
I think.